### DISCUSSION TITLE
如何显示ip地址及标签旁的图标

### DISCUSSION THREAD

**Discussion by YangguangZhou** [walinejs/waline#1934] [Q&A]
![Screenshot_20230416-142124.png](https://user-images.githubusercontent.com/61733195/232276607-fe218877-4fa5-4e0f-8de7-45a6430be7c8.png)
如图

**Comment by lizheming** [comment]
默认就是展示的

**Reply by YangguangZhou** [reply]
我这里并没有显示 ![Screenshot_20230416-143508.png](https://user-images.githubusercontent.com/61733195/232277298-9f68971c-85f2-4e7c-97b0-bbfed99bc3f4.png)

**Reply by lizheming** [reply]
demo 页面忽略，自己网站里需要加一下 https://unpkg.com/@waline/client@2.15.0/dist/waline-meta.css 这个 CSS 样式表

**Reply by YangguangZhou** [reply]
加上后成了这样🥲
![Screenshot_20230416-182416.png](https://user-images.githubusercontent.com/61733195/232297052-388cf4f4-0c47-4b73-a32e-6fdb2f00106b.png)

**Reply by YangguangZhou** [reply]
另外，现在似乎不能正确识别Android14

**Reply by lizheming** [reply]
没有添加 Waline 默认样式表吧，自己多研究下。系统识别我们使用的是 ua-parser.js 库，欢迎给对方 PR。

**Reply by YangguangZhou** [reply]
已解决，谢谢
请问该如何添加IP地址呢
另外，Android无法识别是因为系统本身接口中还没更改版本号

**Reply by YangguangZhou** [reply]
现在又无法正常使用了😮‍💨
![Screenshot_20230416-202437.png](https://user-images.githubusercontent.com/61733195/232309793-d7d2bc05-aec0-4992-abfd-abb4ae650713.png)

环境变量检查了没有问题，leancloud也没问题，使用之前deploy的就是正常的

**Reply by lizheming** [reply]
下午发布的新版本有点问题，已修复，重新更新下试试

**Reply by YangguangZhou** [reply]
可以了，谢谢🙏
但我这里并没有显示ip地址啊

**Reply by lizheming** [reply]
我的时间有限，并不负责手把手指导。请先自行研究一下，如果折腾不出来可以等待其他好心人回复。
